main.js

```diff
@@ -0,0 +1,3 @@
+const suma = (a,b) => {
+    return a + b;
```

Comment:
What's happen if we have a error??
```suggestion
if(!a || !b) return 'Porfa checa tu suma';
    return a + b;
```